src/main/java/org/jalasoft/BankClient.java

```diff
@@ -25,9 +25,14 @@ public void run() {
         bankServices = new BankService();
         scanner = new Scanner(System.in);
         while (!done) {
-            System.out.print("Enter command (0=quit, 1=new, 2=select, 3=deposit, 4=loan, 5=show, 6=interest): ");
+            System.out.print("Enter command (0=quit, 1=new, 2=select, 3=deposit, 4=loan, 5=show, 6=interest, 7=Withdraw): ");
             int commandNumber = scanner.nextInt();
-            processCommand(commandNumber);
+            
+            try {
+                processCommand(commandNumber);
+            } catch (IllegalArgumentException exception) {
```

Comment:
We still need to add ArithmeticException  here :)

Reply:
I put it in the Bank Service line 104. Is it necessary to put again the exception or should I move that exception to that code line?

Reply:
got it, it is ok :)